### PR TITLE
Patch for CVE-2021-29022 and CVE-2021-29023

### PR DESCRIPTION
**Pull request by seran**
Pull Request Checklist

  * [X ] My code follows the code formatting guidelines.
  * [ ] I have an issue ID for this pull request. 
  * [ ] I selected the corresponding branch.
  * [ ] I have rebased my changes on top of the corresponding branch.
  
Issue Type (Please check one or more)

  * [X ] Bugfix
  * [ ] Improvement of an existing Feature 
  * [ ] New Feature

### CODE REVIEW

**Comment by UnderDogg** [comment]
References issue #743

**Comment by UnderDogg** [comment]
@seran thank you for the patch. It looks great!
For the full solution we're looking into not letting the user send the password reset request over and over again, putting a rate limiting on it.

What you've made, however is a great start.

File: application/modules/upload/controllers/Upload.php [hunk 1]
```diff
@@ -122,7 +122,7 @@ public function show_files($url_key, $customerId = null)
                     $obj['name'] = substr($file, strpos($file, '_', 1) + 1);
                     $obj['fullname'] = $file;
                     $obj['size'] = filesize($path . '/' . $file);
-                    $obj['fullpath'] = $path . '/' . $file;
+					$obj['fullpath'] = $file; // Full path removed to as the patch for CVE-2021-29022
```
Comment (zeitschlag) [inline application/modules/upload/controllers/Upload.php:125]: A few weeks ago we decided to get rid of the entire filepath #739. This PR was already merged into `develop`.
It seems to me, that you want to merge into `master`, maybe that's the issue?

Thanks nevertheless for this contribution!

Comment (seran) [reply application/modules/upload/controllers/Upload.php:125]: Make sense, then we don't need to merge this.

**Pull request closed without merging**